bukkit/latest/src/main/java/io/github/wysohn/triggerreactor/bukkit/manager/trigger/share/api/placeholder/PlaceholderExpansionSupport.java

```diff
@@ -93,15 +94,31 @@ public String getVersion() {
      */
     @Override
     public String onPlaceholderRequest(Player player, String identifier) {
```

Comment:
The problem is still there for the latest bukkit. Please test before submitting it!

Reply:
oh it is. sry I was almost drained at that time due to lack of sleep